quickannotator/api/v1/annotation/utils.py

```diff
@@ -1,22 +1,63 @@
 #%%
 import threading
-import numpy as np
 import ray
 import time
 import os
 from datetime import datetime
-
+from quickannotator.db.fsmanager import fsmanager
 from quickannotator.db import get_session
 from quickannotator.db.crud.annotation import build_export_filepath
 from quickannotator.db.crud.annotation import AnnotationStore
 from quickannotator.dsa_sdk import DSAClient
-import geojson
+from quickannotator.db import db_session
 from quickannotator import constants
-from quickannotator.db.crud.image import get_image_by_id
+from quickannotator.db.crud.image import get_image_by_id, add_image_by_path
 from itertools import product
+from quickannotator.constants import IMPORT_ANNOTATION_BATCH_SIZE
 from quickannotator.db.logging import LoggingManager
+import ujson
+from quickannotator.db.crud.annotation_class import get_annotation_class_by_name_case_insensitive
+from quickannotator.db.crud.tile import TileStoreFactory, TileStore
+from tqdm import tqdm
+from shapely.geometry import shape
+
+def import_annotations(image_id: int,annotation_class_id: int, annot_file_path):
+    store = AnnotationStore(image_id, annotation_class_id, is_gt=True)
+    import_geojson_annotation_file(image_id, annotation_class_id, isgt=True, filepath=annot_file_path)
+
+def import_geojson_annotation_file(image_id: int, annotation_class_id: int, isgt: bool, filepath: str):
+    '''
+    This is expected to be a geojson feature collection file, with each polygon being a feature.
+    
+    '''
+    # use ujson to read fast
+    with open(filepath, 'r', encoding='utf-8') as file:
+        # Load the JSON data into a Python dictionary
+        # TODO need to switch to orjson after export annotation and landing page merged.
+        data = ujson.loads(file.read())
+        features = data["features"]
 
 
+    tile_store: TileStore = TileStoreFactory.get_tilestore()
+    annotation_store = AnnotationStore(image_id, annotation_class_id, isgt, in_work_mag=False)
```

Comment:
This makes it seem like the import_annotations() method is completely unnecessary. Is that the case?